elfo-core/src/context.rs

```diff
@@ -105,6 +106,30 @@ impl<C, K> Context<C, K> {
         ward!(self.actor.as_ref().and_then(|o| o.as_actor())).set_status(status);
     }
 
+    /// Gets the actor's status kind.
+    ///
+    /// # Example
+    /// ```
+    /// # use elfo_core as elfo;
+    /// # fn exec(ctx: elfo::Context) {
+    /// // if actor is terminating.
+    /// assert!(ctx.status_kind().is_terminating());
+    /// // if actor is alarming.
+    /// assert!(ctx.status_kind().is_alarming());
+    /// // and so on...
+    /// # }
+    /// ```
+    /// # Panics
+    /// - Panics when called on pruned context.
```

Comment:
1. Add an empty line before this section
2. What's the point to use a list here (-)?

Reply:
Done in [1070425](https://github.com/elfo-rs/elfo/pull/133/commits/1070425102a5c11c57e2f0617259ec6dbfb6c18c).
list is just a silly habit.